package.json

```diff
@@ -6,8 +6,7 @@
   "scripts": {
     "start": "webpack-dev-server --env=dev",
     "build": "webpack --env=prod",
-    "build:test": "webpack --env=test",
-    "test": "mocha --compilers js:babel-core/register -C --recursive"
+    "test": "webpack --env=test && mocha build/test.bundle.js"
```

Comment:
Yeah, we already had webpack setup to handle transpiling; this just applies the webpack configuration to bundling the tests instead of the application.